### PR TITLE
Fixes #197: Log in Sign up UI updated.

### LINKED ISSUE #197: UI improvement for sign up and login screen.
### Would you like to work on the issue?
Yes

### PR DESCRIPTION
**Pull request by mayank408**
whatsapp image 2016-10-16 at 12 31 16 am

### CODE REVIEW

**Comment by the-dagger** [comment]
There is a merge conflict in the manifest. @mayank408

**Comment by mayank408** [comment]
@the-dagger merge conflict resolved.

File: app/src/main/java/org/fossasia/susi/ai/adapters/recyclerAdapters/ChatFeedRecyclerAdapter.java [hunk 1]
```diff
@@ -333,18 +334,19 @@ private void handleItemEvents(final MapViewHolder mapViewHolder, final int posit
         final String toSpeak = model.getContent();
         String[] parts = toSpeak.split(":");
         final String string1 = parts[0];
-        if(MainActivity.checkSpeechOutputPref()){
```
Comment (the-dagger) [inline app/src/main/java/org/fossasia/susi/ai/adapters/recyclerAdapters/ChatFeedRecyclerAdapter.java]: We need to check whether the user has enabled the speech output from susi or now, so don't remove this check.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -37,11 +37,18 @@
...
-            android:label="@string/app_name"
+            android:theme="@style/AppTheme.NoActionBar"
             android:windowSoftInputMode="stateHidden|adjustResize"/>
         <activity
             android:name=".activities.LoginActivity"
-            android:windowSoftInputMode="stateHidden|adjustResize"/>
+            android:windowSoftInputMode="stateHidden|adjustResize"
+            android:theme="@style/AppTheme.NoActionBar"/>
+
+            <intent-filter>
```
Comment (gogeta95) [inline app/src/main/AndroidManifest.xml]: @the-dagger intent filter is supposed to be inside activity tag. this is invalid.

**Pull request merged**